Stream_HomeWork_1/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```

Comment:
Andrey explained what .gitignore is. Binary files (.class, see below) should not be a part of pr as well.